src/main/java/ev3dev/sensors/Battery.java

```diff
@@ -67,38 +67,37 @@ private Battery() {
         currentRereader = new DataChannelRereader(batteryPath + "/" + batteryEv3 + "/" + current);
     }
 
-    public int getVoltageMicroVolts() {
+    public int getVoltageMicroVolt() {
```

Comment:
This changes public API

Reply:
Hmmm, nevermind, this was likely accepted in the previous pull request.

Reply:
I fixed the new method name to match the existing unit postfix.